feat(documentation): adding info regarding image released (#58)
newrelic/nri-kubernetes: README.md

Check out our [documentation](https://docs.newrelic.com/docs/kubernetes-integration-new-relic-infrastructure) in order to find out more how to install and configure the integration, learn what metrics are captured and how to view them.

> Note that `nri-kubernetes` is released as an integration and included into the `infrastructure-k8s` image [available in DockerHub](https://hub.docker.com/r/newrelic/infrastructure-k8s/tags?page=1&ordering=last_updated). 
> This image uses as base one [the image](https://github.com/newrelic/infrastructure-bundle) `infrastructure-bundle` containing most of the integration available and the infrastructure agent used to send data to newrelic.

Reviewer: ```suggestion
Note that `nri-kubernetes` is released both as separate integration and included into the `infrastructure-k8s` image [available in DockerHub](https://hub.docker.com/r/newrelic/infrastructure-k8s/tags?page=1&ordering=last_updated). 
This image uses [`infrastructure-bundle`](https://github.com/newrelic/infrastructure-bundle) as a base, which also most of the integration available and the infrastructure agent used to send data to newrelic.
```

> Note that `nri-kubernetes` is released both as separate integration and included into the `infrastructure-k8s` image [available in DockerHub](https://hub.docker.com/r/newrelic/infrastructure-k8s/tags?page=1&ordering=last_updated). 
> This image uses [`infrastructure-bundle`](https://github.com/newrelic/infrastructure-bundle) as a base, which also most of the integration available and the infrastructure agent used to send data to newrelic.
> 

Reviewer: ```suggestion
Note that `nri-kubernetes` is released both as separate integration and as a part of the `infrastructure-k8s` image, which is [available in DockerHub](https://hub.docker.com/r/newrelic/infrastructure-k8s/tags?page=1&ordering=last_updated). Said image uses [`infrastructure-bundle`](https://github.com/newrelic/infrastructure-bundle) as a base, which additionaly includes the infrastructure agent and most of the available integrations used to send data to newrelic.

```
